src/_guides/language/language-tour.md

```diff
@@ -1136,25 +1136,31 @@ The <code>=> <em>expr</em></code> syntax is a shorthand for
 <code>{ return <em>expr</em>; }</code>. The `=>` notation
 is sometimes referred to as _arrow_ syntax.
 
-<div class="alert alert-info" markdown="1">
-**Note:**
-Only an *expression*—not a *statement*—can appear between the arrow
-(=\>) and the semicolon (;). For example, you can’t put an [if
-statement](#if-and-else) there, but you can use a [conditional
-expression](#conditional-expressions).
-</div>
+<aside class="alert alert-info" markdown="1">
+  **Note:**
+  Only an *expression*—not a *statement*—can appear between the arrow
+  (=\>) and the semicolon (;). For example, you can’t put an [if
+  statement](#if-and-else) there, but you can use a [conditional
+  expression](#conditional-expressions).
+</aside>
```

Comment:
Does dart.dev support {{site.alert.info}} / {{site.alert.end}}  ? That’s what we’re using now on flutter.dev. And, yes, the content is indented by 2 chars.  :)  Also, the macros add the **Note:** or **Warning** text, as the case may be.  (Depending on whether it’s .info or .warning, for example.)

Reply:
Nope. I'll file a bug about that.